System-Automation-Scripts/Amazon-price-tracker/price_tracker.py

```diff
@@ -1,90 +1,130 @@
-#Imports and dependencies
+# Imports and dependencies
 
-import requests
+import datetime
+import sys
 import time
+
+import requests
 from bs4 import BeautifulSoup
 from mail_in_python import send_confirmation
-import datetime  
-    
-#This variable is to test the feasibility of the product
-feasible = False
 
-#This function is to indicate to the user the currency the product is mentioned in, so the same currency is used for budget.
+# This variable is to test the feasibility of the product
+feasible = False
 
-def currency_used(URL):
 
-    #These headers are user specific, look for "my user agent" in the google search bar and replace your user agent here"
-    headers = {"User-Agent": 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'} 
+# This function is to indicate to the user the currency the product is mentioned in,
```

Comment:
Change to ->

`This function is to indicate the user, the currency the product is mentioned in...`